lib/components/molecules/HeroBanner/HeroBanner.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+/**
+ *
+ * HeroBanner
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './HeroBanner.style';
+import type { Props } from './types';
+
+const HeroBanner = ({
+  className,
+  children,
+  heroImageWidth,
+  childrenContainerWidth,
```

Comment:
Instead of taking a width you should allow taking a class or a styled Object. Someone can do more things with it.